packages/orbit-components/src/Textarea/Textarea.stories.tsx

```diff
@@ -7,185 +8,114 @@ import { SPACINGS_AFTER } from "../common/consts";
-  placeholder: "Placeholder",
-  help: "Everything is fine.",
+export const WithHelp: Story = {
+  args: {
+    help: "Everything is fine.",
+    helpClosable: false,
+  },
 };
 
-export const WithError = ({ value, placeholder, error }) => {
-  return (
-    <Textarea placeholder={placeholder} error={error} onChange={action("change")} value={value} />
-  );
+export const WithError: Story = {
+  args: {
+    error: "Something went wrong.",
+  },
 };
 
-WithError.story = {
-  name: "With error",
+export const Controlled: Story = {
+  args: {
+    defaultValue: "defaultValue",
+    value: undefined,
+  },
 
   parameters: {
-    info: "Some description about this type of textarea in general.",
+    info: "Controlled version of Textarea component. Check Orbit.Kiwi for more detailed guidelines.",
+    controls: {
+      exclude: ["onChange", "value"],
+    },
   },
 };
 
-WithError.args = {
-  value: "Something",
-  placeholder: "Placeholder",
-  error: "Something went wrong.",
-};
-
-export const Playground = ({
-  label,
-  value,
-  fullHeight,
-  placeholder,
-  help,
-  error,
-  rows,
-  required,
-  disabled,
-  readOnly,
-  maxLength,
-  resize,
-  dataTest,
-  spaceAfter,
-}) => {
-  return (
-    <Textarea
-      label={label}
-      value={value}
-      fullHeight={fullHeight}
-      placeholder={placeholder}
-      help={help}
-      error={error}
-      rows={rows}
-      required={required}
-      disabled={disabled}
-      readOnly={readOnly}
-      maxLength={maxLength}
-      resize={resize}
-      onChange={action("change")}
-      onFocus={action("focus")}
-      onBlur={action("blur")}
-      dataTest={dataTest}
-      spaceAfter={spaceAfter}
-    />
-  );
-};
-
-Playground.story = {
+export const Playground: Story = {
   parameters: {
-    info: "Some description about this type of textarea in general.",
+    info: "Playground of Textarea component. Check Orbit.Kiwi for more detailed guidelines.",
+    controls: {
+      exclude: ["onChange", "onFocus", "onBlur"],
+    },
   },
-};
 
-Playground.args = {
-  label: "Label",
-  value: "",
-  fullHeight: true,
-  placeholder: "Placeholder",
-  help: "",
-  error: "Something went wrong.",
-  disabled: false,
-  readOnly: false,
-  resize: RESIZE_OPTIONS.VERTICAL,
-  maxLength: Infinity,
-  dataTest: "test",
-  rows: 3,
-  required: false,
-  spaceAfter: SPACINGS_AFTER.MEDIUM,
-};
-
-Playground.argTypes = {
-  resize: {
-    options: Object.values(RESIZE_OPTIONS),
-    control: { type: "select" },
+  args: {
+    ...WithLabel.args,
+    ...WithHelp.args,
+    ...WithError.args,
+    id: "textarea-id",
+    fullHeight: true,
+    placeholder: "Placeholder",
+    disabled: false,
+    readOnly: false,
+    resize: RESIZE_OPTIONS.VERTICAL,
+    maxLength: Infinity,
+    rows: 3,
+    required: false,
+    spaceAfter: SPACINGS_AFTER.MEDIUM,
+    tabIndex: 0,
+    dataAttrs: {
+      "data-story-example": "Storybook-playground",
+    },
+    onChange: action("onChange"),
```

Comment:
The reason why `defaultValue` and `value` are not in the same component is that I'm getting error in case I mix it. That's the reason why I added another story (Controlled, now it's **UNCONTROLLED**)
<img width="447" alt="image" src="https://github.com/user-attachments/assets/9526a095-d7dd-4df0-a3de-a1b92ab429fc">

I could write a comment before the CR, sorry for that. I'll add a note into `info` field.

Reply:
Yeah, I realised that as I went further into the story 😮‍💨